docs/RELEASING.md

```diff
@@ -1,29 +1,27 @@
-# Releasing the Connectors project
+# Releasing the connector service
 
-The version scheme we use is **MAJOR.MINOR.PATCH.BUILD** and stored in the [VERSION](https://github.com/elastic/connectors-ruby/blob/main/VERSION) file
-at the root of this repository.
+> Note: This is for internal use within Elastic. Only Elastic members can release the connector service.
+
+The version scheme we use is **MAJOR.MINOR.PATCH.BUILD** and stored in the [VERSION](https://github.com/elastic/connectors-ruby/blob/main/VERSION) file at the root of this repository.
 
 ## RubyGem Account
 
-When releasing Gems, you will be asked for an Email and Password.
-Look into the Vault in the `ent-search-team/rubygem` secret.
+When releasing Gems, you will be asked for an Email and Password. Look into the Vault in the `ent-search-team/rubygem` secret.
 
 ## Unified release
 
-**MAJOR.MINOR.PATCH** should match the Elastic and Enterprise Search version it targets
-and the *BUILD* number should be set to **0** the day the Connectors release is created
-to be included with the Enterprise Search distribution.
+**MAJOR.MINOR.PATCH** should match the Elastic and Enterprise Search version it targets and the *BUILD* number should be set to **0** the day the Connector service release is created to be included with the Enterprise Search distribution.
 
 For example, when shipping for `8.1.2`, the version is `8.1.2.0`.
 
-To release Connectors:
+To release the connector service:
 
-- Set the VERSION file to the new/incremented version on the release branch
-- Make sure all tests and linter pass with `make lint test`
-- PR these changes to the appropriate Connectors release branch
-- Run `make release`
+1. Set the [VERSION](https://github.com/elastic/connectors-ruby/blob/main/VERSION) file to the new/incremented version on the release branch
+2. Make sure all tests and linter pass with `make lint test`
+3. PR these changes to the appropriate connector service release branch
+4. Run `make release_service release_utility`
```

Comment:
```suggestion
1. Make sure all tests and linter pass with `make lint test`
2. Run `make release_service release_utility`
3. Set the [VERSION](https://github.com/elastic/connectors-ruby/blob/main/VERSION) file to the new/incremented version on the release branch
4. PR these changes to the appropriate connector service release branch
```

Reordered. We should release the current VERSION (was a snapshot, this makes it a release), then increment the version to the new SNAPSHOT.

Reply:
Nice catch @seanstory , I just changed the bullets to numbers without paying attention to the content.